utxonursery.go

```diff
@@ -871,15 +872,14 @@ func (u *utxoNursery) graduateClass(classHeight uint32) error {
 		// generated a sweep txn for this height. Generate one if there
 		// are kindergarten outputs or cltv crib outputs to be spent.
 		if len(kgtnOutputs) > 0 {
-			sweepInputs := make([]sweep.Input,
-				len(kgtnOutputs))
+			sweepInputs := make([]sweep.Input, len(kgtnOutputs))
 			for i := range kgtnOutputs {
 				sweepInputs[i] = &kgtnOutputs[i]
 			}
 
 			finalTx, err = u.cfg.Sweeper.CreateSweepTx(
-				sweepInputs, classHeight)
-
+				sweepInputs, sweepTxConfTarget, classHeight,
```

Comment:
thoughts on passing this in as a nursery config option, instead of accessing a global variable?

Reply:
SGTM, added in the latest diff.